x-pack/plugins/monitoring/public/application/hooks/use_table.ts

```diff
@@ -83,22 +82,18 @@ export function useTable(storageKey: string) {
 
   // get initial state from localStorage
   const [sorting, setSorting] = useState<Sorting>(storageData.sort || { sort: {} });
-  const cleanSortingData = (sortData: Sorting) => {
```

Comment:
this new function was causing the sorting to store itself incorrectly.  it doesn't exist in the angular version and after removal it stopped breaking